linux_os/guide/auditing/auditd_configure_rules/audit_rules_networkconfig_modification_hostname_file/rule.yml

```
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Record Events that Modify the System''s Network Environment'
```

I think it would be great to add the specific file network environment file name to the rule title because now we have multiple rules with the same title which can be confusing sometimes when looking to report.

```suggestion
title: 'Record Events that Modify the System''s Network Environment - /etc/hostname'
```